### PR TITLE
Travisdouce/multiple remotes

### PR DESCRIPTION
**Pull request by tdouce**
Allow 'client' can create test data in two different 'remotes'

### CODE REVIEW

**Comment by tdouce** [comment]
@gschorkopf @iamvery Can you review this PR if you have a few moments? :cherries:

File: spec/integration/remote_factory_girl_spec.rb [hunk 1]
```diff
@@ -49,6 +49,41 @@
       RemoteFactoryGirl.config.https = true
       expect(RemoteFactoryGirl.config.https).to eq true
     end
+
+    context 'when configuring multiple remotes' do
+
+      before do
```
Comment (gschorkopf) [inline spec/integration/remote_factory_girl_spec.rb:56]: This before block is duplicated a few times. Could you break out the configure part into a spec_helper method? Something like:

``` ruby
def configure_remote_factory(host:, port:, end_point:, return_response_as: :as_hash, return_with_root: true)
  RemoteFactoryGirl.configure(:travis) do |config|
    config.home = {:host => host, :port => port, :end_point => end_point }
    config.return_response_as = return_response_as
    config.return_with_root = return_with_root
  end
end

# In your spec!
before do
  configure_remote_factory(host: 'localhost', port: 3000, end_point: '/remote_factory_girl/travis/home')
end
```

**Comment by gschorkopf** [comment]
Lookin' good! I'm sure Jay will have some more productive things to say as well.

File: spec/integration/remote_factory_girl_spec.rb (hunk 1 above)
Comment (tdouce) [inline spec/integration/remote_factory_girl_spec.rb:56]: Yes!. That is a very good idea. :+1:

**Pull request merged**